slices/stunnel4.yaml

```diff
@@ -0,0 +1,30 @@
+package: stunnel4
+
+slices:
```

Comment:
There are some slices missing. At first glance, we're missing `/etc/*/*` (which could be multiple slices, with one of them being `systemd` IMO, which would also include `/lib/systemd/*` and `./usr/lib/systemd/user/stunnel@.service`).

Reply:
`./usr/lib/sysusers.d/stunnel4.conf` is also missing

Reply:
This topic is a bit tricky.

By default, the stunnel4 configuration and the systemd service don't do anything.

```
$ sudo systemctl start stunnel4

$ sudo systemctl status stunnel4
● stunnel4.service - LSB: Start or stop stunnel 4.x (TLS tunnel for network daemons)
     Loaded: loaded (/etc/init.d/stunnel4; generated)
     Active: active (exited) since Mon 2023-06-26 12:17:50 IST; 13s ago
       Docs: man:systemd-sysv-generator(8)
    Process: 88697 ExecStart=/etc/init.d/stunnel4 start (code=exited, status=0/SUCCESS)
        CPU: 8ms
```

The `stunnel4` package needs to be manually configured for it to be useful.

The idea behind opening this PR was to enable folks to consume an existing stunnel slice (which this PR provides) by combining it with their own specific configuration (`stunnel4.conf`).

As such perhaps the `stunnel4` slice shouldn't have a default configuration (which isn't useful) and systemd related bits in it.

I might be missing something in my logic here - thanks for the feedback!